src/lib/Components/Inbox/Conversations/Previews/PDFPreview.tsx

```diff
@@ -56,5 +63,6 @@ export default Relay.createContainer(PDFPreview, {
 interface RelayProps {
   pdfAttachment: {
     file_name?: string
+    download_url?: string
```

Comment:
😞 

Seems so imposible for an attachement to not actually have a `download_url`, no?